content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,17 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< note >}}
+This is an exception for enforcing resourcequota on ephemeral-storage.
```

Comment:
space in resourcequota

Reply:
@gmannos Done !

Reply:
@gmannos I have modified the content in the note as suggested in [comment](https://github.com/kubernetes/website/pull/35011#discussion_r940922995) for wrapping it up . Please take a look again.

Thanks!

Reply:
When reviewing, aim to follow our [style guide](https://kubernetes.io/docs/contribute/style/style-guide/), which I think would recommend.

```markdown
There is an exception for enforcing ResourceQuota on ephemeral storage.
```